content/en/tracing/profiler/enabling/ddprof.md

```diff
@@ -89,22 +102,16 @@ exec ./ddprof --environment prod --service my-web-app --service_version 1.0.3 my
 {{< /tabs >}}
 
 
-5. A minute or two after starting your application, your profiles appear on the [Datadog APM > Profiler page][5].
-
-## Library installation
+5. A minute or two after starting your application, your profiles appear on the [Datadog APM > Profiler page][3].
```

Comment:
Is "a minute or two" still accurate in the contemporary product?  I know that some APM customers complain because it can take ~10 minutes to get the first trace.  Want to be sure we set expectations properly in the average case.

(FWIW, I usually only have to wait a minute or two, just want to make sure that's what our customers can reliably expect)

Reply:
Looking at lag time, we are indeed up to 4 minutes (when everything is OK). The usual wait time will be around 2/3 minutes.

Reply:
It is still very relevant to tell the user to wait for a few minutes.